### PR TITLE
Landing Page and Immuno conclusion dropdown

### PR DESCRIPTION
**Pull request by pascagihozo**
# Pull Requests Requirements

- [ x] The PR title includes a brief description of the work done, including the
      Issue number if applicable.
- [ x] The PR includes a video showing the changes for the work done.
- [ ] The PR title follows conventional commit label standards.
- [ x] The changes confirm to the OpenElis Global x3 Styleguide and design
      documentation.
- [ ] The changes include tests or are validated by existing tests.
- [ x] I have read and agree to the Contributing Guidelines of this project.

## Summary
This pull request includes the landing page that allows the user to select the lab units before they access the common dashboard and it adds a drop down on Immuno-chemistry conclusion
## Screenshots
Screenshot from 2024-08-22 18-20-00
Screenshot from 2024-08-22 18-00-16

[Screencast from 2024-08-22 18-21-01.webm](https://github.com/user-attachments/assets/3a319e42-e8e4-4c00-bb72-2791a480ae65)

## Related Issue
-User expressed a need to select department immediately after they log into the system
-At Immunochemistry lab they require ICD code which should be uniform to pathology conclusion and Immunochemistry conclusion report

## Further work
Features are currently implemented to the frontend and they are yet to be implemented with the backend logic as well

@mozzy11 @caseyi

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -209,10 +216,25 @@ export default function App() {
             <Layout onChangeLanguage={onChangeLanguage}>
               <Switch>
                 <Route path="/login" exact component={() => <Login />} />
+                <Route
```
Comment (mozzy11) [inline frontend/src/App.js:219]: i dont think we should have this route

File: frontend/src/App.js [hunk 2]
```diff
@@ -209,10 +216,25 @@ export default function App() {
...
+                  path="/landing"
+                  exact
+                  component={() => <LandingPage />}
+                />
+                <Route
+                  path="/dashboard"
```
[the remark this answers is not in the document]
Comment (pascagihozo) [reply frontend/src/App.js:225]: The Landing page component rendered on the /landing isn't a part of the Home component. The landing serves the selection of the department(unit/lab) while the Home renders the HomeDashboard

**Review by pascagihozo — COMMENTED** [review]
Please look into it and let me know what you think can be a resolution!

File: frontend/src/App.js (hunk 1 above)
Comment (mozzy11) [reply frontend/src/App.js:212]: Thats fine

**Pull request merged**